packages/account/src/Components/sent-email-modal/sent-email-modal.jsx

```diff
@@ -50,13 +62,38 @@ const SentEmailModal = ({ identifier_title, is_open, onClose, onClickSendEmail }
                     { identifier_title }
                 );
                 break;
+            case 'Change_Email':
+                subtitle = localize('Check your email and click the link in the email to proceed.');
+                break;
             default:
                 subtitle = localize('Please click on the link in the email to reset your password.');
                 break;
         }
         return subtitle;
     };
 
+    const onLiveChatClick = () => {
+        onClose();
+        window.LiveChatWidget?.call('maximize');
+    };
+
+    const live_chat = has_live_chat ? (
```

Comment:
previously we did not have live chat button, I added it to the desired `SendEmail` modal regarfing design